x-pack/plugins/security_solution/public/common/components/threat_match/and_badge.tsx

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
It looks like this one is identical to `x-pack/plugins/security_solution/public/common/components/exceptions/builder/and_badge.tsx` - could we just move these up to /common? Could also be follow up.

Reply:
Yeah, let's do some collapsing as a follow up. There's still a lot going on with this component and backend work I need to take care of that some of this "hard forking" is just me saving short term time.